tools/topology/topology2/platform/intel/sdw-dmic-generic.conf

```diff
@@ -30,11 +30,21 @@ Object.Pipeline {
 			Object.Widget.host-copier.1 {
 				stream_name	"sdw dmic"
 				pcm_id 4
-				Object.Base.audio_format.1 {
-					# 32 -> 16 bits conversion is done here,
-					# so in_bit_depth is 32 (and out_bit_depth is 16).
-					in_bit_depth	32
-				}
```

Comment:
someone will also have to explain to me what this means?

Reply:
> someone will also have to explain to me what this means?

We want to do 32 -> 16 bits conversion in the copier. So, set in_bit_depth	to 32, and the default value of out_bit_depth is 16.